redaxo/bin/console

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+
+set_time_limit(0);
+
+unset($REX);
+$REX['REDAXO'] = true;
+$REX['HTDOCS_PATH'] = '../';
+$REX['BACKEND_FOLDER'] = 'redaxo';
+
+chdir(dirname(__DIR__));
+
+// initialize core
+require __DIR__.'/../src/core/boot.php';
+
+// force debug mode to enable output of notices/warnings and dump() function
+rex::setProperty('debug', true);
+
+$application = new rex_console_application();
+
+rex::setProperty('console', $application);
+
+// initialize packages
+require rex_path::core('packages.php');
+
+$input = new ArgvInput();
+
+$application->run($input);
```

Comment:
Mir stellt sich die die Frage, ob wir hier in der Datei noch zu viel Logik haben?
Denn die Datei liegt außerhalb des Bereichs, der bei Updates aktualisiert wird.
Aus dem Grund enthalten die beiden index.php-Dateien zum Beispiel extrem wenig Logik, die setzen nur die `$REX`-Var und laden die boot.php.
Möglichkeiten:
1. Ähnlich handlen, mit `$REX['CONSOLE'] = true` (statt `$REX['LOAD_PAGE'] = true` wie in den index.php-Dateien) und den Rest die boot.php übernehmen lassen. Also insbesondere auch das Erstellen der Applikation.
2. Zumindest das Setzen der Properties in die `rex_console_application` ziehen. Wobei es so richtig da eigentlich nicht unbedingt rein gehört. Plus eventuell auch noch `new ArgvInput()`.
3. So lassen und hoffen, dass sich so schnell nichts Relevantes ändert.
4. So lassen und das Update so ändern, dass diese Datei auch immer ausgetauscht wird, obwohl sie eigentlich außerhalb des Update-Bereichs liegt.

Wenn wir nicht 4. nehmen, müssen wir auch noch in die update.php des Cores aufnehmen, dass bei Updates von <=5.3 die console-Datei initial angelegt wird.